draft-ietf-webpush-vapid.md

```diff
@@ -270,29 +285,21 @@ subscription to application servers that are able to provide proof of possession
 for the corresponding private key.
 
 This public key is then added to the request to create a push subscription as
-described in {{key}}.  The Crypto-Key header field includes exactly one public
-key.  For example:
+described in {{key}}.  The Crypto-Key header field includes exactly one ECDSA
```

Comment:
I'm not sure how to interpret this - does it include "exactly one public key of type ECDSA" (but maybe other public keys of different types), or exactly one pubkey, of type ecdsa ? 

Would it make sense to relax this ? 
Use case 1: key rotation, where an app would authorize 2 keys. The 'old' key needs to keep working while the changes are propagated.
Use case 2: other schemes that may use public keys. If vapid requires only one ecdsa key - it would require a different header
for future extensions.